Task 2/Task2.1.2/Line.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Task2._1._2
+{
+    class Line:Figure
+    {
+        private int x1, x2, y1, y2;
```

Comment:
```suggestion
        private Point startPoint;
        private Point endPoint;

```
Тут неплохо было бы ввести отдельный класс для точки, т.к. им можно будет и центр с вершинами для фигур описывать.

Reply:
Тогда было бы неплохо и остальные классы переделать.